modules/ROOT/pages/7.0-release-notes.adoc

```diff
@@ -134,6 +134,17 @@ For information on using premium skins and icon packs, see: xref:premium-skins-a
 
 // CCFR here.
 
+=== The toolbar width was miscalculated for the inline editor positioned inside a scrollable container.
+// #TINY-10581
+
+Previously in {productname}, the inline editor's toolbar width was calculated using the `outerContainer` width.
+
+Consequently, the toolbar was given a width of `0`, resulting in the toolbar collapsing to the minimum width.
```

Comment:
here actually the problem was that only in some cases the calculation of the `outerContainer` width results in a `0`

Reply:
```suggestion
Previously, the inline editor's toolbar width was calculated using the `outerContainer` width.

As a consequence, in some cases, the calculation of the `outerContainer` width resulted in a width of `0`, causing the toolbar to collapse to the minimum width.
```

Reply:
@lorenzo-pomili Please re-review the suggested change and approve it your happy with this so i can merge.
Thanks,